dist/palindrom-dom.js

```diff
@@ -2233,7 +2232,11 @@ var Palindrom = (function() {
 
     const proxifiedObj = this.jsonPatcherProxy.observe(
       true,
-      this.filterChangedCallback.bind(this)
+      // JSONPatcherProxy passes a single operation, and handleLocalChange expect a patch (array)
```

Comment:
The above comment is not true anymore. `handleLocalChange` expects an operation.